Paco-iOS/Paco.xcodeproj/project.pbxproj

```diff
@@ -1548,7 +1548,7 @@
 				ORGANIZATIONNAME = Paco;
 				TargetAttributes = {
 					319771C3163F0733000AC06B = {
-						DevelopmentTeam = J7M7JW942W;
+						DevelopmentTeam = 6EDP567RFN;
```

Comment:
For the xcodeproj files, we want to move to a neutral Developer profile so that there isn't constant churn on credentials in this file. For now, since it is only credential changes, please leave this file out of the pull request.